src/style.css

```diff
@@ -0,0 +1,73 @@
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(fonts/Avenir-Book.ttf);
+  src: url(fonts/Avenir-Heavy.ttf);
+  src: url(fonts/Avenir.ttc);
+}
+
+* {
+  box-sizing: border-box;
+  font-family: Avenir, Helvetica, sans-serif;
+  font-weight: bold;
+}
+
+.form {
+  position: relative;
+  margin-top: 20px;
+}
+
+.input_bar {
+  width: 100%;
+  border: 1px solid #e1e7ed;
```

Comment:
Use CSS variables for repeatable color. 

[More Info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)